### ISSUE TITLE
L'IHM plante sur un certain modèle Python

### ISSUE THREAD

**Issue by adumasphi** [persalys/persalys#194]
In GitLab by @mbaudin47 on Mar 29, 2019, 20:52

J'ai maladroitement défini le modèle Python:
```
def _exec(X0,X0):
    return Y
```
puis j'ai cliqué sur la fenêtre de droite "Inputs", et ça a fait le même bruit qu'une GLI-F4:
```
(base) C:\Users\c61372\AppData\Local\Continuum\anaconda3-5-3\Scripts>C:\Users\c61372\AppData\Local\Continuum\anaconda3\Lib\site-packages\otguibase\otgui.exe
WRN - The configuration file has not been found, using default parameters.
terminate called after throwing an instance of 'OT::InvalidArgumentException'
  what():  Two inputs can not have the same name.
```

**Comment by adumasphi** [comment]
In GitLab by @aurelieladier on Apr 1, 2019, 15:46

mentioned in commit b52d6d87976779bc33f2ce9d8053342ff58a5e66

**Comment by adumasphi** [comment]
In GitLab by @aurelieladier on Apr 2, 2019, 10:15

mentioned in commit bd0608c8a4a371750d3aa3a912d0acbe343f061c

**Comment by adumasphi** [comment]
In GitLab by @jschueller on Apr 2, 2019, 11:19

closed via commit e4b857238ca926fc9622c5a883fb909a4926bec5

**Comment by adumasphi** [comment]
In GitLab by @jschueller on Apr 2, 2019, 11:19

closed via merge request !262

**Comment by adumasphi** [comment]
In GitLab by @aurelieladier on Apr 2, 2019, 11:19

closed via commit bd0608c8a4a371750d3aa3a912d0acbe343f061c

**Comment by adumasphi** [comment]
In GitLab by @jschueller on Apr 4, 2019, 11:40

mentioned in commit e4b857238ca926fc9622c5a883fb909a4926bec5
